components/sensor/honeywellabp.rst

```diff
@@ -0,0 +1,69 @@
+Honeywell ABP Pressure Sensors
+==============================
+
+.. seo::
+    :description: Instructions for setting up Honeywell ABP Pressure sensors
+    :image: honeywellabp.jpg
+    :keywords: Honeywell ABP
+
+The ``honeywellabp`` sensor platform allows you to use your Honeywell ABP 
+(`website <https://sps.honeywell.com/us/en/products/sensing-and-iot/sensors/pressure-sensors/board-mount-pressure-sensors/basic-abp-series>`__,
+`datasheet <https://prod-edam.honeywell.com/content/dam/honeywell-edam/sps/siot/en-us/products/sensors/pressure-sensors/board-mount-pressure-sensors/basic-abp-series/documents/sps-siot-basic-board-mount-pressure-abp-series-datasheet-32305128-ciid-155789.pdf?download=false>`__,
+`Mouser <https://www.mouser.ca/new/honeywell/honeywell-abp-pressure-sensors/>`_) pressure and temperature sensors with ESPHome. The :ref:`SPI <spi>` is
+required to be set up in your configuration for this sensor to work
+
+.. figure:: images/honeywellabp.jpg
+    :align: center
+    :width: 50.0%
+
+    Honeywell ABP Pressure and Temperature Sensor.
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    sensor:
+    - platform: honeywellabp
+        pressure:
+            name: "Honeywell pressure"
+            unit_of_measurement: "psi"
+            min_pressure: 0
+            max_pressure: 15
+        temperature:
+            name: "Honeywell temperature"
+        cs_pin: GPIO5 #D5
+        update_interval: 2s
+
+Configuration variables:
+------------------------
+
+min_pressure and max_pressure are from the device datasheet. These are used to calculate the pressure reading published by the sensor.
+
+- **pressure** (*Optional*): The information for the pressure sensor.
+
+  - **name** (**Required**, string): The name for the pressure sensor.
+  - **min_pressure** (**Required**, int or float): Minumim pressure for the pressure sensor.
+  - **max_pressure** (**Required**, int or float): Maximum pressure for the pressure sensor.
+  - **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor for use in lambdas.
+  - **unit_of_measurement** (*Optional*, string): Defaults to ``psi``
```

Comment:
```suggestion
```

Ah I see so the above is in psi. Putting it here makes it more confusing though IMO, can you just add it above like `Minimum pressure for the pressure sensor in psi`.

Reply:
I added some text clarifying the unit of measurements and the default units for minimum and maximum pressure. Hope that makes things more clear. Thanks again for the comments.